query/tests/it/storages/system/settings_table.rs

```diff
@@ -34,6 +34,7 @@ async fn test_settings_table() -> Result<()> {
         "| name                           | value   | default | level   | description                                                                                        | type   |",
         "+--------------------------------+---------+---------+---------+----------------------------------------------------------------------------------------------------+--------+",
         "|                                |         |         |         |                                                                                                    |        |",
+        "| client_enable_async_insert     | 0       | 0       | SESSION | Whether the client open async insert mode, default value: 0                                        | UInt64 |",
```

Comment:
Maybe we should just take `enable_async_insert` as the client setting, server defaults to open async insert.